onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -203,6 +203,10 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
               "The value to be filled in the attention mask. Default value is -10000.0f",
               AttributeProto::FLOAT,
               OPTIONAL_VALUE)
+        .Attr("softmax_scale",
```

Comment:
>softmax_scale [](http://example.com/codeflow?start=15&length=13)

Could you please add same attribute for QAttention

Reply:
sure.
question: do we need this attribute for QOrderedAttn? it seems to me this input might cover the scale: https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/contrib_ops/cuda/quantization/qordered_ops/qordered_attention.cc#L222